src/SFA.DAS.EmployerAccounts.Web/Extensions/UrlHelperExtensions.cs

```diff
@@ -22,6 +22,14 @@ public static string EmployerCommitmentsAction(this UrlHelper helper, string pat
             return AccountAction(helper, baseUrl, path);
         }
 
+        public static string ReservationsAction(this UrlHelper helper, string path)
+        {
+            var configuration = DependencyResolver.Current.GetService<EmployerAccountsConfiguration>();
```

Comment:
Ouch.  Recommend against the service locator can the config not be passed in as an argument?